sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -188,33 +190,37 @@ def _secondary_hostname(self):
         If not available this will be None. To explicitly specify a secondary hostname, use the optional
         `secondary_hostname` keyword argument on instantiation.
 
+        :return: The hostname of the secondary endpoint.
         :type: str or None
         """
         return self._hosts[LocationMode.SECONDARY]
 
     @property
-    def api_version(self):
+    def api_version(self) -> str:
         """The version of the Storage API used for requests.
 
+        :return: The Storage API version.
         :type: str
         """
-        return self._client._config.version  # pylint: disable=protected-access
+        return self._client._config.version  # type: ignore # pylint: disable=protected-access
 
 
 class TablesBaseClient(AccountHostsMixin):
     """Base class for TableClient
 
+    :ivar str table_name: The name of the table.
+    :ivar str scheme: The scheme component in the full URL to the Tables account.
```

Comment:
Still missing:
```
:ivar str url: The storage endpoint.
:ivar str api_version: The service API version.
```
These are defined by the properties declared above on lines 142 and 199

Reply:
Found a mistake: `table_name` is a "ivar" of its child class.